pkg/Microsoft.ML.Experimental/Microsoft.ML.Experimental.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Microsoft.ML.Experimental contains extension method for MLContext to access internal function to cancel ML pipeline.</PackageDescription>
```

Comment:
I assume there are more things we are going to want to add to this package - so this description seems a little to specific to me.